runner/src/main.rs

```diff
@@ -134,6 +117,28 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
     Ok(())
 }
 
+fn match_cmd(opt: &Opt) -> Step {
+    match opt.cmd {
+        Command::RunExamples(ref opt) => run_examples(opt),
+        Command::RunFunctionsExamples(ref opt) => run_functions_examples(opt),
+        Command::BuildFunctionsExample(ref opt) => build_functions_example(opt),
+        Command::BuildServer(ref opt) => build_server(opt, vec![]),
+        Command::BuildFunctionsServer(ref opt) => build_functions_server(opt, vec![]),
+        Command::RunTests => run_tests(),
+        Command::RunCargoTests(ref opt) => run_cargo_tests(opt),
+        Command::RunBazelTests => run_bazel_tests(),
+        Command::RunTestsTsan => run_tests_tsan(),
+        Command::RunCargoFuzz(ref opt) => run_cargo_fuzz(opt),
+        Command::Format(ref commits) => format(commits),
+        Command::CheckFormat(ref commits) => check_format(commits),
+        Command::RunCi => run_ci(),
```

Comment:
This could lead to an infinite loop if someone adds `run-ci` to the CI yaml. Perhaps add a comment somewhere to highlight this in case we forget in future.

Reply:
Hu, I hadn't thought of this! Thank you for catching this! I added a comment to ci.yaml!